opentelekomcloud/resource_opentelekomcloud_s3_bucket_object.go

```diff
@@ -91,6 +91,12 @@ func resourceS3BucketObject() *schema.Resource {
 				Computed:     true,
 			},
 
+			"sse_kms_key_id": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
```

Comment:
This is set to computed, but I never see a `d.Set` with it.

Reply:
It already have the d.set in code line 189.  
https://github.com/zhongjun2/terraform-provider-opentelekomcloud-1/blob/166ea18eb393f5d99c5af4a0512a8eebdd31fe23/opentelekomcloud/resource_opentelekomcloud_s3_bucket_object.go#L189

Reply:
sorry, done,  https://github.com/zhongjun2/terraform-provider-opentelekomcloud-1/blob/add_kms/opentelekomcloud/resource_opentelekomcloud_s3_bucket_object.go#L243